contracts/stake/src/multitest/quick_unbond.rs

```diff
@@ -202,82 +193,17 @@ fn control_case() {
     run_checks(suite);
 }
 
-#[test]
-fn quick_unbond_case() {
-    // same as control case, but quick unbond with no waiting
-    let mut suite = initial_setup();
-
-    suite.quick_unbond(UNBONDER, &[VOTER2, VOTER3]).unwrap();
-
-    suite
-        .distribute_funds(
-            REWARDS_DISTRIBUTOR,
-            REWARDS_DISTRIBUTOR,
-            Some(cash().with_balance(800u128)),
-        )
-        .unwrap();
-
-    run_checks(suite);
-}
-
-#[test]
-fn unbonder_permission_check() {
-    let mut suite = initial_setup();
-
-    assert_eq!(
-        ContractError::Unauthorized {},
-        suite
-            .quick_unbond(VOTER2, &[VOTER2])
-            .unwrap_err()
-            .downcast()
-            .unwrap(),
-        "only unbonder should be able to quick unbond"
-    );
-
-    // now without unbonder
-    let mut suite = SuiteBuilder::new().with_admin(ADMIN).build();
-
-    assert_eq!(
-        ContractError::Unauthorized {},
-        suite
-            .quick_unbond(ADMIN, &[VOTER1])
-            .unwrap_err()
-            .downcast()
-            .unwrap(),
-        "no one should be able to quick unbond"
-    );
-}
```

Comment:
here we do quick_unbond, that's not longer used and we are supposed to verify the error `ContractError::Unauthorized`

unlike [quick_unbond](https://github.com/WHELP-project/whelp-contracts/blob/69301d542681ca84c9e25852efcbbf7183e4dd93/contracts/stake/src/contract.rs#L625) the fn [unbond](https://github.com/WHELP-project/whelp-contracts/blob/69301d542681ca84c9e25852efcbbf7183e4dd93/contracts/stake/src/contract.rs#L573) does not check if the caller is authorized

should we change that?